dev/tools/backport-pr.sh

```diff
@@ -50,6 +50,15 @@ else
 
 fi
 
+if [[ $(git diff HEAD ${BRANCH} -- *.mli) ]]; then
+    echo
+    read -p "Some mli files are modified. Bypass? [y/N] " -n 1 -r
+    echo
+    if [[ ! $REPLY =~ ^[Yy]$ ]]; then
```

Comment:
Does this crash if the character I type is a space or a newline or something active like !?  I'd suggest quoting REPLY to be on the safe side.

Reply:
>Does this crash if the character I type is a space or a newline or something active like !? I'd suggest quoting REPLY to be on the safe side.

`[[` does not require quoting in this case.